### PR TITLE
[pack] additional logging

### PR DESCRIPTION
**Pull request by soninaren**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs [hunk 1]
```diff
@@ -38,13 +38,15 @@ internal class WorkerFunctionMetadataProvider : IWorkerFunctionMetadataProvider
             _environment = environment;
             _channelManager = webHostRpcWorkerChannelManager;
             _workerRuntime = _environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionWorkerRuntime);
+            _logger.LogDebug("WorkerFunctionMetadataProvider initialized with workerRuntime:{workerRuntime}", _workerRuntime);
```
Comment (liliankasem) [inline src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs:41]:
```suggestion
            _logger.LogDebug("WorkerFunctionMetadataProvider initialized with workerRuntime: {workerRuntime}", _workerRuntime);
```

File: src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs [hunk 2]
```diff
@@ -38,13 +38,15 @@ internal class WorkerFunctionMetadataProvider : IWorkerFunctionMetadataProvider
...
         }
 
         public ImmutableDictionary<string, ImmutableArray<string>> FunctionErrors
            => _functionErrors.ToImmutableDictionary(kvp => kvp.Key, kvp => kvp.Value.ToImmutableArray());
 
         public async Task<FunctionMetadataResult> GetFunctionMetadataAsync(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment, bool forceRefresh)
         {
+            _logger.LogDebug("GetFunctionMetadataAsync called with workerRuntime:{workerRuntime}", _workerRuntime);
```
Comment (liliankasem) [inline src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs]:
```suggestion
            _logger.LogDebug("GetFunctionMetadataAsync called with workerRuntime: {workerRuntime}", _workerRuntime);
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -315,265 +315,4 @@ jobs:
       Verbosity: 'Information'
   - publish: $(Build.Repository.LocalPath)\WebJobs.Script.Performance.App
     artifact: Performance
-
-- job: RunUnitTests
```
Comment (liliankasem) [inline azure-pipelines.yml:319]: Did you intentionally check this in? For testing? I'm assuming this is not a change we want to merge

Comment (soninaren) [reply azure-pipelines.yml:319]: This PR is just for testing in private stamp. Going to close this one and open a new

Comment (soninaren) [reply azure-pipelines.yml:319]: have made this a draft PR

File: src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs (hunk 1 above)
Comment (mathewc) [reply src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs:41]: Also for things like type/method names, use nameof so if types are renamed the logs aren't out of sync.

File: src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs [hunk 4]
```diff
@@ -38,13 +38,19 @@ internal class WorkerFunctionMetadataProvider : IWorkerFunctionMetadataProvider
...
+            _logger.LogDebug("WorkerFunctionMetadataProvider initialized with workerRuntime:{workerRuntime}", _workerRuntime);
         }
 
         public ImmutableDictionary<string, ImmutableArray<string>> FunctionErrors
            => _functionErrors.ToImmutableDictionary(kvp => kvp.Key, kvp => kvp.Value.ToImmutableArray());
 
-        public async Task<FunctionMetadataResult> GetFunctionMetadataAsync(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment, bool forceRefresh)
+        public async Task<FunctionMetadataResult> GetFunctionMetadataAsync(IEnumerable<RpcWorkerConfig> workerConfigs, bool forceRefresh)
         {
+            _logger.LogInformation("Before GetFunctionMetadataAsync called with workerRuntime:{workerRuntime}", _workerRuntime);
```
Comment (mathewc) [inline src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs:49]: can use nameof for methods as well

**Comment by mathewc** [comment]
Is this just for debugging or do you intend to actually check this in ultimately? No description given in the PR so it's unclear

**Comment by soninaren** [comment]
> Is this just for debugging or do you intend to actually check this in ultimately? No description given in the PR so it's unclear

This is just for debugging. I am going to close this and open a new PR with proper description and issue attached. Apologies for not providing anything in description. It was meant to stay open only for an hour or so. But we ran into issues testing the PR and it ended up staying open this long.

**Pull request closed without merging**